frontend/src/app/event/widgets/event-detail-card/event-detail-card.widget.ts

```diff
@@ -34,4 +41,18 @@ export class EventDetailCard {
         // Open a snackbar to alert the user
         this.snackBar.open("Event link copied to clipboard.", "", {duration: 3000})
     }
-}
\ No newline at end of file
+
+    /** Delete the given event object using the Event Service's deleteEvent method
+     * @param event: Event representing the updated event
+     * @returns void
+     */
+    deleteEvent(event: Event): void {
+        let confirmDelete = this.snackBar.open("Are you sure you want to delete this event?", "Delete");
+        confirmDelete.onAction().subscribe(() => {
+            this.eventService.deleteEvent(event).subscribe(() => {
+            this.snackBar.open("Event Deleted", "", { duration: 2000 })
+            location.reload();
```

Comment:
Not for this PR and already discussed offline - but this is for @KrisJordan in case he sees, we will attempt to implement an `RxEvent` object to remove this extra page reload. We decided that it would be good not to block all other further progress for this one optimization.